.gitignore

```diff
@@ -1,4 +1,6 @@
 app-config.local.yaml
+docker-compose.yml
```

Comment:
Why is`docker-compose.yml` in .gitignore

Reply:
because `docker-compose.yml` is not needed for the API deploy as well as local development.
`docker-compose` for local development will be available in the mono repository later